src/cargo/core/compiler/build_runner/mod.rs

```diff
@@ -422,6 +426,21 @@ impl<'a, 'gctx> BuildRunner<'a, 'gctx> {
         self.files().metadata(unit)
     }
 
+    /// Returns the list of SBOM output file paths for a given Unit.
+    ///
+    /// Only call this function when `sbom` is active.
+    pub fn sbom_output_files(&self, unit: &Unit) -> CargoResult<Vec<PathBuf>> {
```

Comment:
I'm uncertain if this function should return a `Vec` or if a single `PathBuf` is actually sufficient.

Reply:
It might output several different crate-types, e.g. `rlib`, `cdylib`.